### PR TITLE
Benefit One Line Descriptions working

### LINKED ISSUE #177: The benefit "one line description" that renders on benefit cards should be pulled from the database.
The "one line description" is currently placeholder text that reads "Benefit Description". The database that contains the benefits data is hosted on Airtable. The app pulls data from by making requests to an API.

screen shot 2018-05-23 at 11 52 07 am

### PR DESCRIPTION
**Pull request by apmcguigan**
Resolves #177

### CODE REVIEW

File: .project [hunk 1]
```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (maxneuvians) [inline .project]: Please add this file to `.gitignore` as it is specific to your IDE

File: .settings/.jsdtscope [hunk 2]
```diff
@@ -0,0 +1,7 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (maxneuvians) [inline .settings/.jsdtscope]: Please add the `.settings/` directory to `.gitignore`, IDE specific.

File: .settings/org.eclipse.wst.jsdt.ui.superType.container [hunk 3]
```diff
@@ -0,0 +1 @@
+org.eclipse.wst.jsdt.launching.JRE_CONTAINER
```
Comment (maxneuvians) [inline .settings/org.eclipse.wst.jsdt.ui.superType.container]: Please add the .settings/ directory to .gitignore, IDE specific.

File: .settings/org.eclipse.wst.jsdt.ui.superType.name [hunk 4]
```diff
@@ -0,0 +1 @@
+Global
```
Comment (maxneuvians) [inline .settings/org.eclipse.wst.jsdt.ui.superType.name]: Please add the .settings/ directory to .gitignore, IDE specific.

File: package.json [hunk 5]
```diff
@@ -6,7 +6,7 @@
     "dev": "node server.js",
     "build": "next build",
     "start": "NODE_ENV=production node server.js",
-    "test": "NODE_ENV=test jest --coverage",
+    "test": "set NODE_ENV=test jest --coverage",
```
Comment (maxneuvians) [inline package.json]: Should just be `NODE_ENV`

File: package.json [hunk 6]
```diff
@@ -47,6 +47,7 @@
     "redux": "^4.0.0",
     "redux-devtools-extension": "^2.13.2",
     "redux-thunk": "^2.2.0",
+    "test": "^0.6.0",
```
Comment (maxneuvians) [inline package.json]: we don't need this package

File: sh.exe.stackdump [hunk 7]
```diff
@@ -0,0 +1,11 @@
+Stack trace:
```
Comment (maxneuvians) [inline sh.exe.stackdump]: Please delete this file

**Review by maxneuvians — COMMENTED** [review]
Couple small items, but really well done for a first time out on Github!  🎉

**Comment by sastels** [comment]
A few extra files crept in there but other than that it looks great!

**Comment by maxneuvians** [comment]
I reran the CI build and it fixed with the new package cache

**Review by maxneuvians — APPROVED** [review]
LGTM 🦑

**Pull request merged**